### PR TITLE
updated definitions and testing files in prog-equiv

### PR DESCRIPTION
**Pull request by dlucanu**

### CODE REVIEW

File: samples/prog-equiv/imp/tests/config.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<tests>
```
Comment (dwightguth) [inline samples/prog-equiv/imp/tests/config.xml:4]: this needs an xml and copyright declaration. see tests/config.xml for an example. Make sure you set the correct copyright years.

File: samples/prog-equiv/streams/tests/config.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<tests>
```
Comment (dwightguth) [inline samples/prog-equiv/streams/tests/config.xml:4]: see above comment

File: samples/prog-equiv/tests/config.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!--
```
Comment (dwightguth) [inline samples/prog-equiv/tests/config.xml:4]: see above comment

**Comment by dlucanu** [comment]
Thanks. I did it.

**Comment by dwightguth** [comment]
Is ktest passing on your machine? looks like some of the tests in this directory failed when run on the server.

**Comment by dlucanu** [comment]
Yes, it does:
$ ktest config.xml --skip pdf
Running initial scripts...
SUCCESS
Kompile the language definitions...(2 in total)
SUCCESS
Running /home/dlucanu/k/samples/prog-equiv/imp/peq.k programs... (4 in total, 0 with input, 4 with output, 0 with error)
Running /home/dlucanu/k/samples/prog-equiv/streams/seq.k programs... (6 in total, 0 with input, 6 with output, 0 with error)
SUCCESS

## SUCCESS

Definitions kompiled: 2 (time: 0'15'' elapsed, 0'27'' CPU)
PDF posters kompiled: 0 (time: 0'0'' elapsed, 0'0'' CPU)
Programs krun: 10 (time: 0'7'' elapsed, 0'32'' CPU)

## Total time: 0'22'' elapsed, 0'59'' CPU

**Comment by dwightguth** [comment]
Okay well I don't have time to debug this just now, so you will have to try to figure out what's going on. I would suggest you verify all the usual factors:
1. run ant clean and then rebuild
2. delete your -kompiled and .k directories and then rekompile
3. get the latest version of the k framework by merging master into your branch, and then retest

**Comment by dlucanu** [comment]
I already did these steps before committing and pushing.

**Comment by dlucanu** [comment]
I did all three steps again, I updated the output files and the result is the same. I see that Jenkins reports that the outputs of some programs tests are empty. I really don't understand what happens.

**Comment by dwightguth** [comment]
Okay. I will investigate this on the Jenkins server today.

**Comment by dlucanu** [comment]
Meanwhile I worked in ub2 branch. Today I tried to sync my fork on master branch I got a conflict. I looked in prog-equiv folder and I noticed that all .out file from programs subfolders were deleted.
Is this a part a solution that fixed it? 
How should I solve the conflict in order to not loose the last changes?
Here is the output of the command

```
$ git --rebase upstream master
Unknown option: --rebase
usage: git [--version] [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p|--paginate|--no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [-c name=value] [--help]
           <command> [<args>]
Dorels-MacBook:k dlucanu$ git pull --rebase upstream master
remote: Counting objects: 725, done.
remote: Compressing objects: 100% (366/366), done.
remote: Total 725 (delta 386), reused 523 (delta 249)
Receiving objects: 100% (725/725), 268.20 KiB | 362 KiB/s, done.
Resolving deltas: 100% (386/386), done.
From https://github.com/kframework/k
 * branch            master     -> FETCH_HEAD
First, rewinding head to replay your work on top of it...
Applying: updated definitions and programs in prog-equiv
Using index info to reconstruct a base tree...
M   samples/prog-equiv/config.xml
M   samples/prog-equiv/streams/seq.k
<stdin>:53: trailing whitespace.
                i isNotModifiedBy (#symStmt ( s )) andBool (#symAExp ( a1 )) 
<stdin>:54: trailing whitespace.
                  isNotModifiedBy (#symStmt ( s )) andBool (#symAExp ( a2 )) 
<stdin>:101: trailing whitespace.
                #symInt ( k0 ) ==Int #symInt ( i02 ) *Int #symInt ( i02 ) 
<stdin>:102: trailing whitespace.
                  andBool #symInt ( i01 ) ==Int #symInt ( i02 ) andBool 
<stdin>:103: trailing whitespace.
                  #symInt ( 1 ) ==Int #symInt ( 2 ) andBool #symInt ( 3 ) 
warning: squelched 75 whitespace errors
warning: 80 lines add whitespace errors.
Falling back to patching base and 3-way merge...
Auto-merging samples/prog-equiv/streams/seq.k
CONFLICT (modify/delete): samples/prog-equiv/config.xml deleted in updated definitions and programs in prog-equiv and modified in HEAD. Version HEAD of samples/prog-equiv/config.xml left in tree.
Failed to merge in the changes.
Patch failed at 0001 updated definitions and programs in prog-equiv
The copy of the patch that failed is found in:
   /Users/dlucanu/k/.git/rebase-apply/patch

When you have resolved this problem, run "git rebase --continue".
If you prefer to skip this patch, run "git rebase --skip" instead.
To check out the original branch and stop rebasing, run "git rebase --abort".

```

**Comment by dwightguth** [comment]
I'm not sure why the .out files have been deleted. This is your definition, so I am fine with you making whatever changes to it you want to make. It's your judgement call.

**Comment by cos** [comment]
I will close this for now.

**Pull request closed without merging**